src/controllers/reviewController.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import Review from "../models/Review";
+
+export const createReview = async (req: Request, res: Response) => {
```

Comment:
This function will be later be putted in post router because according to RestFul routes. The reason I leave it here is because Vien has not done the post router.